src/TinyIoC/TinyIoC.cs

```diff
@@ -4125,7 +4246,7 @@ public static Assembly Assembly(this Type type)
     }
 #endif
     // reverse shim for WinRT SR changes...
-#if (!NETFX_CORE && !NETSTANDARD1_5)
+#if (!NETFX_CORE && !PORTABLE && !NETSTANDARD1_0 && !NETSTANDARD1_1 && !NETSTANDARD1_2 && !NETSTANDARD1_3 && !NETSTANDARD1_4 && !NETSTANDARD1_5 || !NETSTANDARD1_6)

```

Comment:
This always evaluates to true, `!NETFX_CORE && !PORTABLE && !NETSTANDARD1_0 && !NETSTANDARD1_1 && !NETSTANDARD1_2 && !NETSTANDARD1_3 && !NETSTANDARD1_4 && !NETSTANDARD1_5` is true when `NETSTANDARD1_6` is defined, and `!NETSTANDARD1_6` is true when `NETSTANDARD1_6` is not defined, so OR-ing them will always evaluate to true 😕 

I think the `||` is supposed to be `&&`.